src/codes.ts

```diff
@@ -8,38 +8,33 @@ class ErrorInvalidCode extends Error {
   }
 }
 
-enum UnspentType {
-  p2sh = 'p2sh',
-  p2shP2wsh = 'p2shP2wsh',
-  p2wsh = 'p2wsh',
-}
+enum EUnspentType { p2sh, p2shP2wsh, p2wsh }
```

Comment:
This way the values will be `1`, `2`, `3`. 

```
[0] % ts-node         
> enum X { a, b, c}
undefined
> X
{ '0': 'a', '1': 'b', '2': 'c', a: 0, b: 1, c: 2 }
```

the name is also a bit awkard, I think `UnspentType` should remain and the tcomb value should be `UnspentTypeTcomb`.

Reply:
Undone.